README.md

```diff
@@ -2,28 +2,29 @@
 
 ### Overview
 
-Easy deploy AI models to Kubernetes using LocalAI, vLLM, DeepSpeed-MII, NVIDIA Triton and custom Docker images.
+Deploy AI models and apps to Kubernetes without hitting every pitfall.
 
-### Key Features
+### Quick Start
 
-- 🌐**Unified Deployment Framework:** Offers a comprehensive set of Kubernetes controllers and custom resource definitions (CRDs) tailored for AI model deployment, eliminating the complexity of managing diverse AI engines.
-- 🛠**Engine Versatility:** While it harmonizes common configuration aspects, Prem Operator respects the unique capabilities of each AI engine, such as vLLM or Triton, facilitating optimal deployment strategies based on engine strengths.
-- 📦**Custom Resource Ecosystem:** Includes specialized resources like AI Deployment Custom Resource, AIModelMap, and AutoNodeLabeler, each with its controller, simplifying the intricate process of AI model deployment and management within Kubernetes environments.
-- 🔄**Seamless Integration:** Designed to integrate smoothly with existing Kubernetes clusters, enhancing the deployment, scaling, and management of AI models without disrupting current operations.
+Deploy the operator and chat with Hermes in 2 or 3 steps. Note that you need a NVIDIA GPU with 8GB of RAM.
 
-### Why Use Prem Operator?
+1. Install the NVIDIA Operator if not present already
```

Comment:
```suggestion
1. Install the NVIDIA Operator if not present already (not needed if the host already has the relevant drivers installed, and is manually managed)
```

Reply:
I am not sure if we should add all this information so a more succint version of this.

Reply:
The operator is at least needed for node labeling unless they label the nodes themselves. It is possible to install the operator without it configuring the runtime and drivers.

I don't really want to get into whether it's possible to use an NVIDIA card without the operator. Especially not in the quickstart example(s).